docs/Documentation/Drivers/Classic/GlobalParam.md

```diff
@@ -131,9 +131,9 @@ This section describes how to define the forcing files needed by the VIC model.
 
 Unlike model parameters, for which 1 file contains data for all grid cells, the meteorological forcings are stored as a separate time series for each grid cell. The time step length of the input forcings must match the time step length at which VIC is running. Input files can be ASCII or Binary (signed or unsigned short ints) column formatted. Columns in the file must be in the same order as they are defined in the global control file.
 
```

Comment:
I'm not able to add line comments on previous lines - for state file section starting from line 121, add `STATESEC`, and in comments for `STATENAME`, add that both the date and second of the state will be appended.